### PR TITLE
Revise JupyterHub docs for JupyterHub revamp

### PR DESCRIPTION
**Pull request by willfurnass**
Still to do:

- [x] Revise https://docs.hpc.shef.ac.uk/en/latest/hpc/jupyterhub/file_browse.html
- [x] Revise https://docs.hpc.shef.ac.uk/en/latest/hpc/jupyterhub/terminal.html
- [x] Revise https://docs.hpc.shef.ac.uk/en/latest/hpc/jupyterhub/conda.html
- [x] Revise https://docs.hpc.shef.ac.uk/en/latest/hpc/jupyterhub/nb_usage.html
- [x] Review https://docs.hpc.shef.ac.uk/en/latest/hpc/jupyterhub/sess_monit.html

### CODE REVIEW

File: hpc/jupyterhub/terminal.rst [hunk 1]
```diff
@@ -3,14 +3,12 @@
...
-the **New** menu within 
-Jupyter's :ref:`Files <jh_file_browse>` tab.
-This is a fully-featured web terminal, 
-running on a worker node.  
-It opens a new browser tab.
+You can start a terminal  
 
-.. image:: /images/jupyterhub/jupyterhub-terminal.png
+* Via the top menu bar (**File**, **New**, **Terminal**) or
+* via the :ref:`Files <jh_jupyterlab>` Launcher tab.
```
Comment (Carldkennedy) [inline hpc/jupyterhub/terminal.rst:9]:
```suggestion
* Via the :ref:`Files <jh_jupyterlab>` Launcher tab.
```

**Review by Carldkennedy — COMMENTED** [review]
LGTM with one small correction

**Comment by willfurnass** [comment]
Re the merge:  the updated service isn't yet live!

**Review by nikishe — COMMENTED** [review]
LGTM

**Pull request merged**